docs/_layouts/post.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+---
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+  <header class="post-header">
+    <h1 itemprop="name headline">{{ page.title | escape }}</h1>
+    <p class="meta">
+        <time datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">
+            {% assign date_format = site.date_format | default: "%b %-d, %Y" %}
```

Comment:
Same here as above for the dates? Can we have a default format for the dates that we use? That we can configure in one place if need be?